src/controllers/invoices.js

```diff
@@ -43,6 +43,9 @@ export default function () {
     const buyer = order.buyers && order.buyers[0]
     const shippingLine = order.shipping_lines && order.shipping_lines[0]
     const transaction = order.transactions && order.transactions[0]
+    const getItemsValid = order.items.filter(function (item) {
+      return item.quantity > 0
+    })
```

Comment:
```suggestion
    const getItemsValid = order.items && order.items.filter(item => item.quantity > 0)
```

Só evitando pau se `order.items` não for Array...